src/components/street-geo.js

```diff
@@ -7,9 +7,12 @@ const MAPBOX_ACCESS_TOKEN_VALUE =
 
 AFRAME.registerComponent('street-geo', {
   schema: {
-    longitude: { type: 'number', default: 0 },
-    latitude: { type: 'number', default: 0 },
-    elevation: { type: 'number', default: 0 },
+    longitude: { type: 'number', default: null },
+    latitude: { type: 'number', default: null },
```

Comment:
Is it okay to change default to null for longitude and latitude now?

Reply:
I don't know. It's probably safest to keep 0